src/ralph/data_importer/management/commands/demodata.py

```diff
@@ -217,703 +221,699 @@ def add_user_and_group(username, password, group_name, permissions):
             user.save()
 
         dc_permissions = [
```

Comment:
do we have to list every possible permission? couldn't we just grant all in particular app or app.model?